src/style.css

```diff
@@ -1,3 +1,74 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+
+  --active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header-logo {
+  display: flex;
+  margin-right: 243px;
+}
+
+@media (min-width: 1024px) {
+  .header {
+    padding: 0 50px;
+  }
+}
+
+@media (min-width: 1200px) {
+  .header-logo {
+    margin-right: 419px;
+  }
+}
+
+.nav-list {
+  list-style: none;
+  display: flex;
+  font-size: 12px;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-item {
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+}
+
+.nav-item:not(:last-child) {
+  margin-right: 20px;
```

Comment:
Be consistent with your vertical margins. It's common practice to add only top or only bottom margin, but not both. This helps to avoid unexpected layout shifts due to margin collapsing. In this case, you're adding a margin to the right, which is horizontal, but it's still good to keep this in mind.